hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonInputFormatBase.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
should we consolidate the interface as following, 
CarbonInputFormatBase
      --> CarbonInputFormat
                  --> CarbonColumnarInputFormat
                  --> CarbonRowInputFormat
or even further merging CarbonInputFormat into CarbonInputFormatBase